### ISSUE TITLE
Sharing from android share menu resets timed message setting to "off"

### ISSUE THREAD

**Issue by carlosdavidepto** [wireapp/wire-android#1703]
## The problem

> Briefly describe the issue you are experiencing (or the feature you want to see added to Wire).

Sharing a file (a picture from the galery, for example) from outside the app via the share menu resets the timed message setting for the conversation to "Off".

> Tell us what you were trying to do and what happened instead.

I was trying to share a file in a conversation set to expire messages in 1 day. I would expect the shared file to have the same setting (i.e. expire in 1d). Instead, as soon as the file is sent to the other party in the conversation, the setting for message expiration for the sent file is reset to "Off" for the shared file and all other subsequent messages until manualy restored to it's previous setting.

Note that this **issue does not occur when sharing is initiated from inside the wire app** itself, such as when you tap the camera icon. **It only occurs when I use the Android share menu outside the app** (such as when sharing a link from a browser).

## Environment

* Wire version that exhibits the issue: **3.14.628**
* Last Wire version that did not exhibit the issue (if applicable): **N/A**
* Android OS version used to run Wire: **6.0.1**
* Is this a custom firmware or a stock one: **Stock**
* Mobile phone model/manufacturer: **Samsung Galaxy S5mini Duos**
* Mobile network type (EDGE/LTE/Wi-Fi/Offline): **Wi-Fi**

## Details

> Describe the steps necessary to reproduce the problem in more details. The detailed description will help us to fix the issue faster.

0. Before starting set one of your conversations to expire messages in one day.
1. Open another app with share menu integration (gallery, browser, etc.)
2. Select one item and tap the share icon
3. From the share menu, select Wire as the target app
4. From the Wire conversation list, select the conversation from step 0
5. Notice that the item from step 2 was sent in the conversation without an expiry. It should instead expire in one day, as per step 0
6. Notice also that all messages sent after this step in that same conversation will also have no expiry, and that the timed message indicator is "Off"

**Comment by someoneEsle** [comment]
duplicate of #1282
